sql-database/managed-instance/create-and-configure-managed-instance.ps1

```diff
@@ -20,16 +23,26 @@ $miAdminSqlPassword = "ChangeYourAdminPassword1"
 $edition = "General Purpose"
 $vCores = 8
 $maxStorage = 256
-$computeGeneration = "Gen4"
+$computeGeneration = "Gen5"
 $license = "LicenseIncluded" #"BasePrice" or LicenseIncluded if you have don't have SQL Server licence that can be used for AHB discount
 
 # Set subscription context
-Set-AzContext -SubscriptionId $subscriptionId 
+Set-AzContext -SubscriptionId $SubscriptionId 
 
 # Create a resource group
-$resourceGroup = New-AzResourceGroup -Name $resourceGroupName -Location $location
+$resourceGroup = New-AzResourceGroup -Name $resourceGroupName -Location $location -Tag @{Owner="SQLDB-Samples"}
 
 # Configure virtual network, subnets, network security group, and routing table
+$networkSecurityGroupMiManagementService = New-AzNetworkSecurityGroup `
+                      -Name 'myNetworkSecurityGroupMiManagementService' `
+                      -ResourceGroupName $resourceGroupName `
+                      -location $location
```

Comment:
Use splatting instead of line continuation.
```suggestion
$parameters = @{
  Name = 'myNetworkSecurityGroupMiManagementService' 
  ResourceGroupName = $resourceGroupName 
  location = $location
}
$networkSecurityGroupMiManagementService = New-AzNetworkSecurityGroup @parameters
```